test/hotspot/jtreg/runtime/cds/appcds/customLoader/ProhibitedPackageNamesTest.java

```diff
@@ -28,7 +28,7 @@
  * @requires vm.cds
  * @requires vm.cds.custom.loaders
  * @library /test/lib /test/hotspot/jtreg/runtime/cds/appcds
- * @compile ClassListFormatBase.java ../test-classes/Hello.java test-classes/InProhibitedPkg.java
+ * @compile ../ClassListFormatBase.java ../test-classes/Hello.java test-classes/InProhibitedPkg.java
```

Comment:
Similar comment for avoiding `@compile`.  `@compile` has been mis-used by older CDS tests. We should avoid using it, and should remove it when we are touching the old tests.